games-misc/katawa-shoujo/katawa-shoujo-1.3.1-r1.ebuild

```diff
@@ -5,8 +5,8 @@ EAPI=6
 inherit eutils gnome2-utils
 
 DESCRIPTION="Bishoujo-style visual novel by Four Leaf Studios"
-HOMEPAGE="http://katawa-shoujo.com/"
-SRC_URI="http://dl.katawa-shoujo.com/gold_1.3.1/%5B4ls%5D_katawa_shoujo_1.3.1-%5Blinux-x86%5D%5B18161880%5D.tar.bz2 -> ${P}.tar.bz2
+HOMEPAGE="https://katawa-shoujo.com/"
```

Comment:
should be https://www.katawa-shoujo.com/ as it doesn't redirect as one would expect.